Source/Core/DiscIO/WbfsBlob.cpp

```diff
@@ -50,41 +52,46 @@ u64 WbfsFileReader::GetDataSize() const
   return WII_SECTOR_COUNT * WII_SECTOR_SIZE;
 }
 
-bool WbfsFileReader::OpenFiles(const std::string& filename)
+void WbfsFileReader::OpenAdditionalFiles(const std::string& path)
 {
-  m_total_files = 0;
+  if (path.length() < 4)
+    return;
 
   while (true)
   {
-    auto new_entry = std::make_unique<file_entry>();
-
     // Replace last character with index (e.g. wbfs = wbf1)
-    std::string path = filename;
-    if (m_total_files != 0)
-    {
-      path[path.length() - 1] = '0' + m_total_files;
-    }
+    std::string current_path = path;
+    if (m_files.size() >= 10)
```

Comment:
`m_files.size()` is always 1 when entering the loop, but moving the check wouldn't be a good idea. The reason I'm checking it here is to avoid getting a strange file name on the next line. AddFileToList shouldn't need to care about the details of how this function generates file names. Also, the IOFile object gets created before calling AddFileToList, and it's cleaner to not attempt to open paths where we expect the file name extension to contain a colon. (Maybe the code would work correctly even with this check completely removed, unless some OS tries to handle colons in a funny way, but I thought it would be better to not rely on that.)